server/server.go

```diff
@@ -80,9 +80,10 @@ type Command struct {
 
 // NewCommand returns a new instance of Main.
 func NewCommand(stdin io.Reader, stdout, stderr io.Writer) *Command {
+	s, _ := pilosa.NewServer()
```

Comment:
I realize that in its current state this can't return a non-nil error because no opts were provided, but we should panic on error here in case the NewServer() function ever changes.

Reply:
very good point